---
result: FAILURE
failure_category: unknown
failure_cause: unknown
timestamp: 2016-07-12 20:34:32 UTC
url: http://manhattan.ci.chef.co/job/chef-test/architecture=x86_64,platform=el-6,project=chef,role=tester/69/
trigger_url: http://manhattan.ci.chef.co/job/chef-trigger-release/60/
duration: 51m21s
logs:
  consoleText:
    682: |2
        /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-12.12.24/spec/support/shared/integration/integration_helper.rb:111
      The new definition will overwrite the original one.
      /opt/chef/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- win32/daemon (LoadError)
        from /opt/chef/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
      Run options:
    9118: |
      rspec ./spec/functional/run_lock_spec.rb:279 # Chef::RunLock when locking the chef-client run test returns without waiting when the lock is acquired

      Build step 'Invoke XShell command' marked build as failure
      Archiving artifacts
      Recording fingerprints
